x-pack/test/observability_functional/apps/observability/alerts/index.ts

```diff
@@ -223,7 +223,9 @@ export default ({ getService }: FtrProviderContext) => {
           const actionsButton = await observability.alerts.common.getActionsButtonByIndex(0);
           await actionsButton.click();
           await observability.alerts.common.viewRuleDetailsButtonClick();
-          expect(await find.existsByCssSelector('[title="Rules and Connectors"]')).to.eql(true);
+          expect(await find.existsByCssSelector('[data-test-subj="ruleDetailPageTitle"]')).to.eql(
```

Comment:
@fkanout Currently there's a problem with current test which keeps failing because rule is undefined and rule details page can not be loaded. Here's where rule is undefined https://github.com/elastic/kibana/blob/main/x-pack/plugins/observability/public/pages/rule_details/index.tsx#L201. Stack Management Rule details page is getting the rule as param https://github.com/elastic/kibana/blob/main/x-pack/plugins/triggers_actions_ui/public/application/sections/rule_details/components/rule_details.tsx#L76 and I guess that's why the old detail page could load in the tests without issues. This refactoring is out of scope of this issue. How do you suggest we proceed with this failing test?  

<img width="1712" alt="Screenshot 2022-05-19 at 08 43 03" src="https://user-images.githubusercontent.com/2852703/169228805-3b6b2481-985e-4cf0-aec0-3ce8cb697ae2.png">

Reply:
I just pushed a fix that checks if Observability is in the breadcrumb https://github.com/elastic/kibana/pull/132479/commits/1f94a1f2bc0e9b93868a3b52b9e3c20de2984cc9#diff-5a75bdfd91813e385e6c322eed601a2fd55d34f4a56b7669b0bb193014df1648R226

Reply:
@mgiota, I think this ok for the purpose of this test and in the scope of this PR.